stance/skeleton/skeleton.py

```diff
@@ -11,8 +14,14 @@ def __init__(self, head: Point, tail: Point) -> None:
         self.head: np.ndarray = np.array(head)
         self.tail: np.ndarray = np.array(tail)
         self.vec = self.head - self.tail
-        self.magnitude = self._magnitude()
-        self.direction = self._direction()
+        self.magnitude = np.linalg.norm(self.vec)
+
+        if self.magnitude == 0:
+            logger.warning("Zero vector has no direction: {}".format(self.__repr__()))
```

Comment:
I think this would be better as a debug, because when I run the program, it seems that it happens quite a bit, and it doesn't matter if it has magnitude 0. Esp for squatting, bc the leg vectors aren't the ones that have mag 0.

Reply:
A floating point divide by 0 error should definitely not be a debug statement. It can cause major problems and undefined behavior. The better thing to do would be to figure out why this is happening in the first place

Reply:
Hmm I guess yeah we can make that a separate issue.